### PR TITLE
Implement bulk deletion for services

### LINKED ISSUE #1743: Service batch deletion
Another part of epic : https://github.com/epinio/epinio/issues/1739

Implement the ability to perform bulk deletions of services from CLI

### PR DESCRIPTION
**Pull request by andreas-kupries**
Fix #1743

### CODE REVIEW

File: internal/cli/delete.go [hunk 1]
```diff
@@ -10,10 +10,19 @@ var ()
...
 // CmdAppDelete implements the command: epinio app delete
 var CmdAppDelete = &cobra.Command{
-	Use:               "delete NAME1 [NAME2 ...]",
-	Short:             "Deletes one or more applications",
-	Args:              cobra.MinimumNArgs(1),
-	ValidArgsFunction: matchingAppsFinder,
+	Use:   "delete NAME1 [NAME2 ...]",
+	Short: "Deletes one or more applications",
+	Args:  cobra.MinimumNArgs(1),
+	ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```
Comment (jimmykarily) [inline internal/cli/delete.go:16]: How is `matchingAppsFinder` different? Also, I see the methods are missing the `DisableVersionWarning()`. A rebase needed?

Comment (andreas-kupries) [reply internal/cli/delete.go:16]: `matchingAppsFinder` completes the first argument of the command as app name. The method here completes any argument as app name - It is missing the restriction to the first argument, the check for `len(args)`.

Yes, rebase is needed. Will do.

Comment (andreas-kupries) [reply internal/cli/delete.go:16]: Done (Rebase, and fix): Commit 1c9a378f99f35c944a48e05991397cc558fb0f8b

**Review by jimmykarily — APPROVED** [review]

**Comment by mmartin24** [comment]
Checked working after release `v1.4.0`

```
➜  ~ epinio service delete svc-1 svc-2

🚢  Deleting Services...
Names: svc-1, svc-2
Namespace: workspace

✔️  Services Removed.
Names: svc-1, svc-2
Namespace: workspace

➜  ~ epinio version
Epinio Version: v1.4.0
Go Version: go1.18.5
```

**Pull request merged**